ionic_user_interface/src/views/Gyms.vue

```diff
@@ -2,28 +2,47 @@
   <ion-page>
     <ion-content :fullscreen="true">
       <div id="container">
-        <strong>Route Maker</strong>
-        <p>Find climbing routes by gym</p>
+        <div v-if="gymNameString === ''">
+          <strong>Route Maker</strong>
+          <p>Find climbing routes by gym</p>
+        </div>
+        <div v-else class="gym-name">
+          <b>-- {{ gymNameString }} --</b>
+        </div>
         <br />
-        <gym-selector v-if="canvasWidth > 0" :width="canvasWidth" />
+        <gym-selector
+          v-if="showGymSelector"
+          :width="canvasWidth"
```

Comment:
Actually what's this canvasWidth thing for?

Reply:
I can't find a good way in css to make the map square. I.e- set the width same as height other than this

Reply:
oh ok

Reply:
Might wanna try some of the solutions here: 
https://stackoverflow.com/questions/5445491/height-equal-to-dynamic-width-css-fluid-layout

Reply:
particularly the one with 467 upvotes

Reply:
yea i tried. so far only my way works

Reply:
You can just use `vw` units, I pushed the fix to this PR